### PR TITLE
Liberapay

### PR DESCRIPTION
**Pull request by TobiGr**
I added Liberpay to donation page and removed some unused code.

@theScrabi I'll update the readme with some new screenshots and the donation hint later today.

### CODE REVIEW

File: css/donate.css [hunk 1]
```diff
@@ -92,26 +58,45 @@ main .row [class^="col"] {
...
+    margin-bottom: 30px;
+}
+.button-wrapper {
+    flex-grow: 1;
+    margin-bottom: 30px;
+}
+#bountysource-donation {
+    text-align: center;
+}
+#librepay-donation {
```
Comment (TheAssassin) [inline css/donate.css]: It's liberapay, but I guess it doesn't _really_ matter.

File: donate/index.html [hunk 2]
```diff
@@ -129,15 +104,26 @@ <h4>Every donation increases our motivation.</h4>
...
+                                <div class="button-wrapper" id="bountysource-donation">
+                                    <a href="https://www.bountysource.com/teams/newpipe/" target="_blank">
+                                        <button class="donate">Bountysource</button>
+                                    </a>
+                                    <div class="clearfix"></div>
+                                    <span>boost an upcoming feature</span>
+                                </div>
+                                <div class="button-wrapper" id="librepay-donation">
+                                    <a href="https://www.bountysource.com/teams/newpipe/" target="_blank">
+                                        <button class="donate">Liberpay</button>
```
Comment (TheAssassin) [inline donate/index.html]: It's Liberapay.

**Comment by TheAssassin** [comment]
Thanks, @TobiGr. I found a few typos, but otherwise, lgtm. I'll deploy as soon as you merge the PR.

**Comment by TobiGr** [comment]
Three different versions of one word 😆 It's too late. xD

File: donate/index.html (hunk 2 above)
Comment (TobiGr) [reply donate/index.html]: Thanks

**Pull request merged**